packages/compat/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "../../tsconfig.json",
+	"compilerOptions": {
+		"rootDir": ".",
+		"noImplicitAny": false,
```

Comment:
As I'm sure you ran into as well, I too would have hoped to be able to turn on this strictness sooner than later, but there are a ton of existing issues to resolve first. I do like that in this implementation, it's straight-forward to opt-out specific packages, so that they can be each be improved over time in isolation.